packages/core/elasticsearch/core-elasticsearch-client-server-internal/src/agent_manager.ts

```diff
@@ -22,6 +23,14 @@ export interface AgentFactoryProvider {
   getAgentFactory(agentOptions?: HttpAgentOptions): AgentFactory;
 }
 
+export interface AgentManagerOptions {
+  /**
+   * The maximum number of seconds to retain the DNS lookup resolutions.
+   * Set to 0 to disable the cache (default Node.js behavior)
```

Comment:
```suggestion
   * Set to 0 to disable the cache (default Node.js behavior)
```

Reply:
@TinaHeiligers, did you plan to change anything in the suggestion? It looks the same to me.

Reply:
I was going to explain why we added the setting then changed my mind to make it a `nit` instead.